Userland/Utilities/nc.cpp

```diff
@@ -144,6 +164,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     fd_set readfds, writefds, exceptfds;
 
+    std::size_t rcv_buff_size = get_max_tcp_rcv_buf_size(max_tcp_rcv_buff_size_input);
```

Comment:
Please don't randomly abbreviate identifiers. We don't have a limit on horizontal length.
Please use `size_t` instead of `std::size_t`.